autoload/vsnip/session/snippet/parser.vim

```diff
@@ -135,27 +135,27 @@ let s:variable2 = s:map(s:seq(s:dollar, s:open, s:varname, s:close), { value ->
       \ } })
 
 " Can't understand this, So currently unsupport.
```

Comment:
I didn't understand the use case of this feature, so I left it unimplemented.
Thank you for implement it.

Could you remove this comment?

Reply:
Ok!